_data/transport.yml

```diff
@@ -177,4 +177,4 @@ websites:
       url: https://www.westjet.com/
       twitter: WestJet
       img: westjet.png
-      tfa: No
+      tfa: No
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.